packages/react-components/src/components/resource-explorers/explorers/alarm-explorer/transformAlarmData.ts

```diff
@@ -0,0 +1,45 @@
+import { AlarmData, AlarmDataStatus } from '../../../../hooks/useAlarms';
+import { parseAlarmStateAssetProperty } from '../../../../hooks/useAlarms/transformers';
+import {
+  AlarmResource,
+  AlarmResourceWithLatestValue,
+} from '../../types/resources';
+
+export const transformAlarmData = ({
+  assetId,
+  assetModelId,
+  compositeModelId,
+  compositeModelName,
+  state,
+  status,
+  source,
+}: AlarmData): {
+  resource: Partial<AlarmResourceWithLatestValue>;
+  status: AlarmDataStatus;
+} => {
+  const latestState = parseAlarmStateAssetProperty(state?.data?.at(-1));
+
+  return {
+    resource: {
+      name: compositeModelName,
+      assetCompositeModelId: compositeModelId,
+      assetId,
+      assetModelId,
+      sourcePropertyId: source?.property.id,
```

Comment:
This should be `inputPropertyName` --> `alarmData.inputProperty[0].name`

We don't need to show the alarm source propertyId

Reply:
oh I see, this is the wrong "source". haha.

Reply:
Yeah this is exactly why I renamed "sourceProperty" to "inputProperty" cause it's easy to mess that up :)